yarn.lock

```diff
@@ -6121,7 +6038,6 @@ upath@^1.1.1:
 uri-js@^4.2.2:
   version "4.2.2"
   resolved "https://registry.yarnpkg.com/uri-js/-/uri-js-4.2.2.tgz#94c540e1ff772956e2299507c010aea6c8838eb0"
-  integrity sha512-KY9Frmirql91X2Qgjry0Wd4Y+YTdrdZheS8TFwvkbLWf/G5KNJDCh6pKL5OZctEW4+0Baa5idK2ZQuELRwPznQ==
```

Comment:
I'm not sure what happened to these. Is the yarn version on our workstations too old? Are these necessary?

Reply:
https://stackoverflow.com/questions/53540429/what-is-the-integrity-property-inside-yarn-lock-file